tests/integ/modin/frame/test_unstack.py

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
+#
+
+import modin.pandas as pd
+import numpy as np
+import pandas as native_pd
+import pytest
+
+from tests.integ.modin.sql_counter import sql_count_checker
+from tests.integ.modin.utils import eval_snowpark_pandas_result
+
+
+@sql_count_checker(query_count=1)
+def test_unstack_input_no_multiindex():
+    # Only 1 Query as input dataframe to unstack is not a multi-index,
```

Comment:
I see a lot of our tests are trying to reset the names of index, do we have any test that originally have a name configured for the name fields?